Fixes DOC-13145 : Updated configure-cloud-org-sso (#19536)
cockroachdb/docs: src/current/cockroachcloud/configure-scim-provisioning.md

<a id="scim-group-push"></a>

### Supported features 

* Create users
* Update user attributes
* Deactivate users
* Import users
* Import groups
* Profile sourcing
* Group push

For Okta idp, user attributes can not be update for Admin users. This is an API limitation.

> For more information on the listed features, visit the [Okta Glossary](https://help.okta.com/okta_help.htm?type=oie&id=ext_glossary).

Reviewer: I believe these can be removed.

- The endpoint to the CockroachDB {{ site.data.products.cloud }} SCIM API, `https://cockroachlabs.cloud/api/scim/v2`.
- The API token of a CockroachDB {{ site.data.products.cloud }} service account with the [**Org Administrator**]({% link cockroachcloud/authorization.md %}#org-administrator) role.

> #### To add SCIM integration using Okta Integration Network (OIN): 

Reviewer 2: ```suggestion

Depending on your setup, you can configure SCIM either via the Okta Integration Network (OIN) for a standardized app or manually for a custom SAML app integration.

### Add SCIM integration using Okta Integration Network (OIN)
```

- The endpoint to the CockroachDB {{ site.data.products.cloud }} SCIM API, `https://cockroachlabs.cloud/api/scim/v2`.
- The API token of a CockroachDB {{ site.data.products.cloud }} service account with the [**Org Administrator**]({% link cockroachcloud/authorization.md %}#org-administrator) role.

> #### To add SCIM integration using Okta Integration Network (OIN) : 

Reviewer: ```suggestion
To add SCIM integration using Okta Integration Network (OIN): 
```

1. Log in to Okta Admin Dashboard as an admin user.
1. Click **Applications** > **Applications** > **Browse App Catalog**
1. Search for **Cockroach Labs** > Click **Add**

> 1. Enter **Application label** > **Entity ID** and **ACS URL** should be "NA" >  Click **Next** > Click **Done**

Reviewer: Shouldn't these values be set, because SCIM requires SAML underneath?

1. Go to **Provisioning** Tab and click **Configure API Integration**
1. Check **Enable API integration** 
1. Provide <b>API authentication token</b>: the API token for a CockroachDB {{ site.data.products.cloud }} <a href="managing-access.html#create-a-service-account">service account</a> with the <a href="authorization.html#org-administrator"><b>Org Administrator</b></a> role > **Test API Credentials**
1. Click **Save**.
1. Click **To App**. This tab controls assignment of Okta identities to CockroachDB {{ site.data.products.cloud }}. To allow provisioning and deprovisioning of users, ensure that **Create Users** and **Deactivate Users** are selected, and make any other desired changes.
1. Optionally, click **To Okta**. This tab allows you to import a CockroachDB {{ site.data.products.cloud }} organization's existing users into Okta. This helps to maintain an updated list of IAM users when an organization creates IAM users in a variety of ways. Refer to Okta's documentation about mapping individual fields. Make any desired changes.

> #### To add SCIM provisioning to a SAML app integration in Okta:

Reviewer: ```suggestion
To add SCIM provisioning to a SAML app integration in Okta:
```
